pkg/oc/bootstrap/docker/openshift/helper.go

```diff
@@ -69,6 +74,10 @@ var (
 		"/sys/fs/cgroup:/sys/fs/cgroup:rw",
 		"/dev:/dev",
 	}
+	registryTrustContainersBinds = []string{
+		"/var/lib/origin:/var/lib/origin:ro",
```

Comment:
Actually we should use the directory that the user specified as the config directory if they used the --host-config-dir flag.